### PR TITLE
Wf60 to Wf62 spec edit recommendations

### PR DESCRIPTION
**Pull request by FaraiMutero**
Wf60 to Wf62 spec edit recommendations

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -170,6 +170,34 @@ paths:
...
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /processes/{processId}/stop:
+    post:
+      tags:
+        - v1
+        - developers
+        - Runtime
+      summary: stops a process execution; deletes a process instance
+      operationId: start
```
Comment (taylordowns2000) [inline api/swagger.yaml]: start or stop?

File: api/swagger.yaml [hunk 2]
```diff
@@ -170,6 +170,34 @@ paths:
...
+      description:
+        Create a process instance and register it with the execution engine
```
Comment (taylordowns2000) [inline api/swagger.yaml]: wrong maybe?

File: api/swagger.yaml [hunk 3]
```diff
@@ -170,6 +170,34 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  processId:
+                    type: integer
+                    description: The unique processe ID.
+        '404':
+          description: Invalid processId. Process does not exist
```
Comment (taylordowns2000) [inline api/swagger.yaml]: This would be a not found, not an "invalid"

**Review by taylordowns2000 — CHANGES_REQUESTED** [review]
maybe a few typos, please see change requests, but otherwise I'm happy. request a review from @aarelaponin please

File: api/swagger.yaml (hunk 1 above)
Comment (taylordowns2000) [reply api/swagger.yaml]: fixed in [caf5c3e](https://github.com/GovStackWorkingGroup/bb-workflow/pull/74/commits/caf5c3ed7a1fb69f4f816ae25ef928378f7294e9)

File: api/swagger.yaml (hunk 2 above)
Comment (FaraiMutero) [reply api/swagger.yaml]: Resolved. Thanks

**Review by taylordowns2000 — APPROVED** [review]

**Review by taylordowns2000 — APPROVED** [review]

**Comment by FaraiMutero** [comment]
Conflicts resolved. @taylordowns2000 please check if changes are pulling through on your end?

**Pull request closed without merging**